plugins/Marketplace/resources/qml/OnboardBanner.qml

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.1
+
+import UM 1.6 as UM
+import Cura 1.6 as Cura
+
+// Onboarding banner.
+Rectangle
+{
+    property bool bannerVisible
+    property string bannerIcon
```

Comment:
This should be an alias instead of a string. See https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#property-aliases

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/0ca20e9b3d0f6716cb3fff68442661375bc712e3